src/main/java/com/dev/cinema/model/dto/MovieSessionRequestDto.java

```diff
@@ -1,16 +1,16 @@
 package com.dev.cinema.model.dto;
 
 import com.sun.istack.NotNull;
-import javax.validation.constraints.Min;
+import javax.validation.constraints.Size;
 
 public class MovieSessionRequestDto {
     @NotNull
     private String showTime;
     @NotNull
-    @Min(1)
+    @Size(min = 1)
```

Comment:
please see the javadoc 
https://docs.oracle.com/javaee/6/api/javax/validation/constraints/Size.html

are wrappers supported?